app/components/Edit/EditSections.js

```diff
@@ -587,10 +596,16 @@ class EditSections extends React.Component {
 
   render() {
     let resource = this.state.resource;
-    let actionButtons = [<button className="edit--aside--content--submit" disabled={this.state.submitting} onClick={this.handleSubmit}>Save changes</button>,
-    <button className="edit--aside--content--deactivate" disabled={this.state.submitting} onClick={() => this.handleDeactivation('resource', resource.id)}>Deactivate</button>];
+    let actionButtons = [
+      <button className="edit--aside--content--button" disabled={this.state.submitting} onClick={this.handleSubmit}>Save changes</button>,
```

Comment:
Can we use consistent capitalization on button text? From looking on other pages, it looks like we generally capitalize every word besides articles like "as".

Reply:
Definitely! Good catch!